binding.gyp

```diff
@@ -29,7 +28,7 @@
             ['OS=="mac" or OS=="solaris"', {
               'xcode_settings': {
                 'GCC_ENABLE_CPP_EXCEPTIONS': 'YES',
-                'MACOSX_DEPLOYMENT_TARGET': '10.9',
+                'MACOSX_DEPLOYMENT_TARGET': '10.15',
```

Comment:
What's the reason for bumping this?

Reply:
I am not on Mac myself but I've been using this target in Zadeh, and I haven't had any complaints from the users. If we could remove this variable and let Node-gyp handle it for us, it would be awesome.

https://github.com/atom-community/zadeh/blob/0a5ca9a129fdfd947fa4c697dfdbbdc36dbd721f/binding.gyp#L26

Reply:
I'll try this locally on my Mac to confirm. Might as well dust off the old zeromq.js set up as well.